### PR TITLE
MAINT: Add license header to all files

### LINKED ISSUE #64: add copyright header to top of all files
See setup.py for the content.

### PR DESCRIPTION
**Pull request by jakereps**
I'd like to thank `sed` for this one... 84 files in less than a few minutes.

```bash
find app/ -type f -name '*.js*' -execdir sed -i -e '1i\
// ----------------------------------------------------------------------------\
// Copyright (c) 2016--, QIIME development team.\
//\
// Distributed under the terms of the Modified BSD License.\
//\
// The full license is in the file LICENSE, distributed with this software.\
// ----------------------------------------------------------------------------\
\
' {} \;
```

Resolves #64

### CODE REVIEW

**Comment by jairideout** [comment]
👍 Thanks @jakereps, nice `find`/`sed` sorcery! @ebolyen can you take a pass and merge?

File: app/css/main.css [hunk 1]
```diff
@@ -1,4 +1,14 @@
 /*
+----------------------------------------------------------------------------
+Copyright (c) 2016--, QIIME development team.
+
+Distributed under the terms of the Modified BSD License.
+
+The full license is in the file LICENSE, distributed with this software.
+----------------------------------------------------------------------------
+*/
```
Comment (ebolyen) [inline app/css/main.css]: Drop the copyright header for this or move the `hljs` stuff to it's own file. (I'd probably do the latter).

File: config/postcss.config.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+// ----------------------------------------------------------------------------
+// Copyright (c) 2016--, QIIME development team.
+//
+// Distributed under the terms of the Modified BSD License.
+//
+// The full license is in the file LICENSE, distributed with this software.
+// ----------------------------------------------------------------------------
```
Comment (ebolyen) [inline config/postcss.config.js]: Delete this file, unless I'm mistaken we don't use postcss yet (we might someday).

**Comment by ebolyen** [comment]
Two comments regarding odd files which probably should be renamed/shouldn't exist.

Nice `sed` sorcery!

**Comment by jakereps** [comment]
Thanks @jairideout and @ebolyen, fixed those two issues, so it should be good to go!

**Comment by ebolyen** [comment]
Thanks @jakereps!

**Pull request merged**